russh-util/src/runtime.rs

```diff
@@ -0,0 +1,16 @@
+use std::future::Future;
+
+pub fn spawn<F>(future: F)
```

Comment:
This needs to be generic over `Output` as well, as it's used in various places. I prefer this PR to include the migration of `russh` and `russh-keys` over to the generic runtime before merging.